### ISSUE TITLE
How to embed unlabeled data in clustering results composed of several labeled data

### ISSUE THREAD

**Issue by sydddl** [xinglab-ai/mda#2]
Due to my research needs, If I want to see the nearest neighbor relationships embedded in my model latent space for the representation of a class (not actually in the model).

I read your article are two types: unsupervised and supervised. I feel that the supervised method with label alignment is better. 

How can I maintain the mda embedding space unchanged and embed some new unlabeled data? Is this possible? Or there can be other compromise methods？

Thanks

**Comment by xinglab-ai** [comment]
Hi, Thanks for raising the issue. Currently, MDA cannot be used in a semisupervised format (some of the data labels are known). However, we are working on it. Please check our Github repo for the updates.

**Comment by sydddl** [comment]
I am sorry, there were some things I failed to explain clearly before. For example，in `sklearn.decomposition` the PCA can `.fit(X)`, then I can use the pca object to transform other data like Y,  and your demo more like `.fit_transform(X)`, no corresponding interface. Other example is UMAP [Transforming New Data with UMAP](https://umap-learn.readthedocs.io/en/latest/transform.html) , But I don’t know whether the supervised method can separate the `fit` and `transform` steps, both of my examples are working under unsupervised conditions, so I raised this issue.

I think this requirement is simpler than semisupervised train, thanks for your reply.

**Comment by xinglab-ai** [comment]
Yes, you are correct. In our update, we will include the fit function .fit_transform(). We are working on it.

**Comment by sydddl** [comment]
> Yes, you are correct. In our update, we will include the fit function .fit_transform(). We are working on it.

🆗，Thank you very much and I look forward to your work
